src/fetchData/makeContextObject.ts

```diff
@@ -46,18 +47,23 @@ export function makeGetInitialPropsContext({
 
 export function makeGetServerSidePropsContext({
   pageObject,
-  options: { req: reqMocker, res: resMocker, previousRoute },
+  options,
 }: {
   pageObject: FoundPageObject;
   options: ExtendedOptions;
 }): GetServerSidePropsContext<typeof pageObject.params> {
+  const { req: reqMocker, res: resMocker, previousRoute } = options;
   const { params, query, resolvedUrl } = pageObject;
   const { req, res } = makeHttpObjects({
     pageObject,
     reqMocker,
     resMocker,
     refererRoute: previousRoute,
   });
+  const { locale, locales, defaultLocale } = makeRouterMock({
```

Comment:
I agree that having a `getLocales` helper would be better, but I also see that `makeRouterMock` doesn't have significative side effects. If extracting locales logic is a mess, can we consider acceptable using `makeRouterMock` out of its context?

BTW great job! I did't remember we supported locales :)

@evocateur is there locale feature we still don't support? If so I'd love to note it down somewhere (readme or code). I ask since I haven't used Next for while now and never with locales.